sadl3/com.ge.research.sadl.parent/com.ge.reasearch.sadl.errorgenerator/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This changes and the two below as well is from the `reaserach` project which we have removed and replaced with `the c.g.research.sadl.errorgenerator`.
Please leave as is, we will get rid of this when we have merged to master. Just wanted to clarify the commit comment. Also, please note, Eclipse will complain when you try to import both the `research` and `reaserch` projects into workspace.